src/tribol/utils/TestUtils.cpp

```diff
@@ -833,7 +833,7 @@ void TestMesh::allocateAndSetVelocities( int meshId, real valX, real valY, real
                   "not a valid mesh id." );
    }
 
-   SLIC_DEBUG_IF( deleteVels, "TestMesh::allocateAndSetVelocities(): " << 
+   SLIC_INFO_IF( deleteVels, "TestMesh::allocateAndSetVelocities(): " << 
```

Comment:
@srwopschall can you verify these changes are OK?  I think `SLIC_DEBUG_IF` seems like the right level here, but #39 will have to be addressed first.